pkg/appliance/change/change.go

```diff
@@ -30,18 +31,23 @@ func (ac *ApplianceChange) RetryUntilCompleted(ctx context.Context, changeID, ap
 	err := backoff.Retry(func() error {
 		change, err := ac.Get(ctx, changeID, applianceID)
 		if err != nil {
+			log.WithError(err).Error("error getting change ID")
 			return err
 		}
 		if change.GetStatus() == "running" {
-			return errors.New("Change is still running, retry")
+			err := errors.New("Change is still running, retry")
+			log.WithError(err).Error("")
```

Comment:
I'd suggest updating this from ERROR to INFO/WARNING, based on the output seen when using  `--ci-mode --no-interactive`

Reply:
ok, fixed as well